packages/react-components/react-nav-preview/src/components/NavSubItemGroup/useNavSubItemGroup.ts

```diff
@@ -15,18 +16,24 @@ export const useNavSubItemGroup_unstable = (
   props: NavSubItemGroupProps,
   ref: React.Ref<HTMLDivElement>,
 ): NavSubItemGroupState => {
+  const { open } = useNavCategoryContext_unstable();
+
+  // const focusableProps = useTabsterAttributes({ focusable: { excludeFromMover: true } });
+  // const navigation = useAccordionContext_unstable(ctx => ctx.navigation);
+
   return {
-    // TODO add appropriate props/defaults
+    open,
     components: {
-      // TODO add each slot's element type or component
       root: 'div',
     },
-    // TODO add appropriate slots, for example:
-    // mySlot: resolveShorthand(props.mySlot),
     root: slot.always(
       getIntrinsicElementProps('div', {
-        ref,
+        // FIXME:
+        // `ref` is wrongly assigned to be `HTMLElement` instead of `HTMLDivElement`
+        // but since it would be a breaking change to fix it, we are casting ref to it's proper type
+        ref: ref as React.Ref<HTMLDivElement>,
```

Comment:
No need for the ref comment and you can remove the typecast since you have the ref type correctly in the params. You should also move the ref after the spread to avoid overriding it.

Reply:
Updated :-)